packages/did/src/Did.chain.ts

```diff
@@ -153,15 +153,9 @@ function decodeDidPublicKeyDetails(
   const key = keyDetails.key.isPublicEncryptionKey
     ? keyDetails.key.asPublicEncryptionKey
     : keyDetails.key.asPublicVerificationKey
-  const keyType = key.type.toLowerCase() as Lowercase<typeof key.type>
-  if (!keyType) {
```

Comment:
No way this will happen

Reply:
true, I wonder if it was meant to be `key.type?.toLowerCase()` ? But even then, the enum type always has a value.